waku/node/waku_node.nim

```diff
@@ -397,6 +408,9 @@ proc mountRelay*(node: WakuNode,
 
   node.switch.mount(node.wakuRelay, protocolMatcher(WakuRelayCodec))
 
+  node.wakuRelay.addDefaultValidator(messageSizeValidator,
```

Comment:
Nitpick. I think is a bit confusing to use `addDefaultValidator`. IMO, when I read this code, I have the feeling that we will have only one default validator but we are keeping an array of default validators.

```suggestion
  node.wakuRelay.addValidator(messageSizeValidator,
```

For the sake of simplicity, would it be possible to stick only to the concept of the "validators" list?

Reply:
So we already have the `addValidator` method which is for pubsub topic specific validators.

https://github.com/waku-org/nwaku/blob/c1121dd1db27344ad38d6ba85c072f0e75eb4e8e/waku/waku_relay/protocol.nim#L182-L186

The concept of the default validator list comes from go-waku, which are the validators that are supposed to run for all pubsub topics. So we need to save in different data structures the validators per pubsub topic and the validators that run for all topics